src/software.jl

```diff
@@ -66,12 +76,32 @@ function raMOInput(
     return raMOInput(xtal, wave, fermi) 
 end
 
-function raMOInput(directory::AbstractString, ::FromVASP)
+function raMOInput(;
+    POSCAR = "POSCAR",
+    WAVECAR = "WAVECAR",
+    KPOINTS = "KPOINTS",
+    OUTCAR = "OUTCAR"
+)
+    return raMOInput(POSCAR, WAVECAR, KPOINTS, OUTCAR, FromVASP())
+end
+
+"""
+    raMOInput(directory, FromVASP(); CONTCAR=false)
+    raMOInput(FromVASP(); CONTCAR=false)
+
+Reads VASP output files required for DFT-raMO from the given directory. If no directory is given,
+the current directory is checked.
+    
+If `CONTCAR` is set to true, the CONTCAR file will be used instead of the POSCAR.
```

Comment:
I'm not quite sure why we would need to read in CONTCARs. DFT-raMO should always use inputs from a static calculation.

Reply:
That's fair - I left it as a keyword argument so it can be ignored.